pages/contribute.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: page
+title: Contribute
+order: 6
+---
+
+MDAnalysis is an open-source project that welcomes contributions from the community. Whether you're a developer, documentation writer, or interested in improving the website, there are many ways to get involved.
+
+You can contribute by **developing new features, fixing bugs, improving documentation, enhancing tutorials, or engaging in discussions**. If you're new, you can start by participating in community conversations and mentoring programs. For more information on how you can participate, check out the [Community]({{ site.baseurl }}/pages/community/) and [Events]({{ site.baseurl }}/pages/events/) pages.
+
+This page provides an overview of how you can contribute to the [Code](#contributing-to-the-code) or [Documentation](#contributing-to-documentation).
+
+MDAnalysis maintains **three GitHub repositories**, each dedicated to a specific area of contribution:
+
+- [Main Code Repository]({{ site.github.repo }}): for code development and API documentation.
+- [User Guide Repository]({{ site.docs.userguide.repo }}): containing installation instructions, tutorials and learning resources.
```

Comment:
This is too narrow. We have many other libraries where we would want help, MDAKits, etc. Can you rewrite to take that into account?

Singling out the MDAnalysis library is ok as this is the focus (see also mission). Mentioning the User Guide is good although you'd might want to make clear what kind of contributions would go there.